scripts/typeGeneration/convert.js

```diff
@@ -0,0 +1,135 @@
+import { basename, dirname, join, resolve } from 'https://deno.land/std/path/posix.ts';
+import { parse, stringify } from 'https://deno.land/std/yaml/mod.ts';
```

Comment:
just for my curiosity does this mean that we are moving towards deno?

Reply:
For me this just means this is quicker to get going (+ I could use typescript here right away without any setup) - there's not that big of a difference, but we avoid the install step for the dependencies...